cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -738,20 +1090,74 @@ spec:
       - ob
       - obs
   scope: Cluster
-  subresources:
-    status: {}
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+      subresources:
+        status: {}
+      schema:
+        openAPIV3Schema:
+          type: object
+          properties:
+            spec:
+              type: object
+              properties:
+                storageClassName:
+                  type: string
+                endpoint:
+                  type: object
+                  nullable: true
+                  properties:
+                    bucketHost:
+                      type: string
+                    bucketPort:
+                      type: integer
+                      format: int32
+                    bucketName:
+                      type: string
+                    region:
+                      type: string
+                    subRegion:
+                      type: string
+                    additionalConfig:
+                      type: object
+                      nullable: true
+                authentication:
```

Comment:
In my prototype I wasn't sure about the schema for the OB authentication. If you can use the preserve unknown settings on this, let's remove the authentication specs to be on the safe side.

Reply:
Done.